DatasourceUnavailableEventMessage:
  datasource_id: ""
  dataset_id: ""
  title: ""
  url: ""
  error_explanation: ""